### PR TITLE
Updates samples

### PR DESCRIPTION
**Pull request by NikitaAvraimov-okta**

### CODE REVIEW

File: browser-sign-in/package.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "lint": "eslint ."
   },
   "dependencies": {
-    "@okta/okta-react-native": "2.1.1",
+    "@okta/okta-react-native": "2.3.0",
     "events": "^3.2.0",
     "react": "17.0.1",
     "react-native": "^0.64.1",
```
Comment (mraible) [inline browser-sign-in/package.json:16]: Is it possible to update these dependencies to their latest releases too?

File: browser-sign-in/package.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
     "lint": "eslint ."
   },
   "dependencies": {
-    "@okta/okta-react-native": "2.1.1",
+    "@okta/okta-react-native": "~2.3.0",
```
Comment (NikitaAvraimov-okta) [inline browser-sign-in/package.json:13]: note

File: custom-sign-in/package.json [hunk 3]
```diff
@@ -11,7 +11,7 @@
   },
   "dependencies": {
     "@okta/configuration-validation": "^1.0.0",
-    "@okta/okta-react-native": "2.1.1",
+    "@okta/okta-react-native": "~2.3.0",
```
Comment (NikitaAvraimov-okta) [inline custom-sign-in/package.json:14]: note

**Review by JayNewstrom — APPROVED** [review]

File: browser-sign-in/package.json (hunk 1 above)
Comment (NikitaAvraimov-okta) [reply browser-sign-in/package.json:16]: Well, it looks like we do not have full 0.65+ support on android yet, we have to update okta-react-native first

**Pull request closed without merging**